### PR TITLE
Add network module

### PR DESCRIPTION
**Pull request by chimp1984**
Based on https://github.com/bisq-network/misq/pull/4

### CODE REVIEW

**Comment by chimp1984** [comment]
Merged dependent PRs and rebased.

**Review by jmacxx — APPROVED** [review]
utACK

File: network/src/main/java/network/misq/network/http/common/ClearNetHttpClient.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                connection.setRequestProperty(header.first(), header.second());
+            });
+
+            if (httpMethod == HttpMethod.POST) {
+                connection.setDoOutput(true);
+                connection.getOutputStream().write(param.getBytes(StandardCharsets.UTF_8));
+            }
+
+            int responseCode = connection.getResponseCode();
+            if (responseCode == 200) {
```
Comment (kaiwolfram) [inline network/src/main/java/network/misq/network/http/common/ClearNetHttpClient.java]: What about other success codes like 204? Same in Tor Client

Comment (chimp1984) [reply network/src/main/java/network/misq/network/http/common/ClearNetHttpClient.java]: Good point. Yes I guess we should consider all 2xx codes as success case.

Comment (ghubstan) [reply network/src/main/java/network/misq/network/http/common/ClearNetHttpClient.java]: Resolved by commit 521de33087c3fbd3d52b651620930e169a1ada8b.

**Comment by ghubstan** [comment]
utACK

**Pull request merged**